modules/36_buildings/simple/equations.gms

```diff
@@ -73,7 +73,7 @@ q36_H2Share(t,regi)..
 ***---------------------------------------------------------------------------
 *'  CES markup cost to represent sector-specific demand-side transformation cost in buildings
 ***---------------------------------------------------------------------------
-q36_costCESmarkup(t,regi,in)$(ppfen_CESMkup_dyn36(in))..
+q36_costCESmarkup(t,regi,in)$(ppfen_buildings_dyn36(in))..
```

Comment:
is this coordinated with Robin?

Reply:
This is already merged in another PR #674.